flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/ProcTimeBoundedRangeOver.scala

```diff
@@ -69,10 +70,10 @@ class ProcTimeBoundedRangeOver(
     output = function.createOutputRow()
```

Comment:
When the code-gen'd functions need as `Row` to set the result. We can simply pass `output.row` then and later emit `output`.